appium/webdriver/appium_service.py

```diff
@@ -119,7 +119,7 @@ def _parse_host(args):
                 return args[idx + 1]
         return DEFAULT_HOST
 
-    def start(self, **kwargs):
+    def start(self, **kwargs: Union[Dict, str, int]) -> sp.Popen:
```

Comment:
to me it does not make much sense to add typing to kwargs

Reply:
Hum, it makes sense, thanks.
(It's unnecessary to maintain return type when new type can be taken by `**kwarg`.